client/src/services/textFormat.js

```diff
@@ -22,6 +22,10 @@ const formatDateString = (dateString) => {
 // and from "EXAM" to "Exam"
 const formatCourseType = (courseType) => {
   switch (courseType) {
+  case 'teaching-participation-lectures':  // Type from Sisu API
```

Comment:
Should we also handle the case of `hyl/tt/ht` here? Such a grading scale is used for e.g. the Swedish language exams taken by Finnish-speaking students.

Reply:
Yes we can add that option. Do we have any documentation about what these types from Sisu could be? For example, is `hyl/tt/ht` confirmed to be a possible response from Sisu?

Reply:
This would fall under grading scale below rather than course instance type, but yes, it would be easy to add that to the server code, once we know the proper term for this scale.

> Do we have any documentation about what these types from Sisu could be?

One way to figure this out would be to try fetching information about a course which does use this scale from the Sisu API. Documentation probably also exists, but I don't know / can't remember where to find it.

Reply:
Right, that should be added under grading scale but we have a similar function for the grading scale on the client side that we have for the course instance type so adding the correct term to the code client shouldn't be a problem either.

Reply:
The Sisu API returns this as `toinen-kotim`. Based on that, I'd add this as an enum value:
```ts
SecondLanguage = 'SECOND_LANGUAGE'
```
Any opinions?

Reply:
That seems to be a good translation, or `'SECOND_NATIVE_LANGUAGE'`. But since this is used to return the values 'hyvät taidot', 'tyydyttävät taidot' etc., I'm not sure if the term used in Sisu is the best. But if we want to follow the example from Sisu, I'd say `'SECOND_LANGUAGE'` or `'SECOND_NATIVE_LANGUAGE'` would work.

Reply:
Here are some terms that came to mind, but I feel like using some kind of `LANGUAGE` type would be fine as well.

Level of understanding
Level of skills
skill level
Level of abilities
Ability
Competence

Reply:
Something needs to be picked so I chose `SECOND_NATIONAL_LANGUAGE`, which can be changed later.

Reply:
I then need to add this to frontend as well. So hold on the review @Gskartwii